### PR TITLE
Alert docs update

### PR DESCRIPTION
**Pull request by cveigt**
### :pushpin: Summary

This PR updates the recommendation to use the secondary standalone link as secondary action in the `Alert` and `Toast`.

Preview: https://hds-website-git-cv-alert-docs-update-hashicorp.vercel.app/components/alert

Screen Shot 2023-06-22 at 9 29 28 AM

Preview: https://hds-website-git-cv-alert-docs-update-hashicorp.vercel.app/components/toast

Screen Shot 2023-06-22 at 9 29 58 AM

### :link: External links

Jira ticket: [HDS-1993](https://hashicorp.atlassian.net/browse/HDS-1993)

***

### 👀 Reviewer's checklist:

- [ ] +1 Percy if applicable
- [ ] Confirm that PR has a changelog update via [Changesets](https://github.com/changesets/changesets) if needed

:speech_balloon: Please consider using [conventional comments](https://conventionalcomments.org/) when reviewing this PR.

[HDS-1993]: https://hashicorp.atlassian.net/browse/HDS-1993?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: website/docs/components/alert/partials/guidelines/guidelines.md [hunk 1]
```diff
@@ -127,12 +127,15 @@ Avoid using critical buttons in alerts. We handle the prominence and importance
 
 #### Links
 
-Use [standalone links](/components/link/standalone) when an action takes the user to a new destination (URL). Follow the standalone link [usage guidelines](https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/HDS-Product---Components?node-id=2522%3A8014&t=s1vlvHztHFnd5T9i-1) to determine what variant “type” to use.
+Use [standalone links](/components/link/standalone) when an action takes the user to a new destination (URL). Follow the standalone link [usage guidelines](https://helios.hashicorp.design/components/link/standalone) to determine what variant “type” to use.
```
Comment (heatherlarsen) [inline website/docs/components/alert/partials/guidelines/guidelines.md:130]: > Follow the standalone link [usage guidelines](https://helios.hashicorp.design/components/link/standalone) to determine what variant “type” to use.

Is this part still necessary since we're now providing them with an official recommendation?

Comment (jorytindall) [inline website/docs/components/alert/partials/guidelines/guidelines.md:130]: I know you didn't change this as part of this work, but since this is referring to a component should it be "Standalone Link"?

File: website/docs/components/toast/partials/guidelines/guidelines.md [hunk 2]
```diff
@@ -84,7 +84,7 @@ We recommend using the `secondary` Button variant for primary actions and the `t
   <T.Title>Recommended button usage</T.Title>
   <T.Description>Lorem ipsum dolar sit amet, consecteu adipiscig elit null dignissim felis.</T.Description>
   <T.Button @text="Button" @color="secondary" />
-  <T.Link::Standalone @color="primary" @icon="plus" @iconPosition="leading" @text="Link text" @href="#" />
+  <T.Link::Standalone @color="secondary" @icon="plus" @iconPosition="leading" @text="Link text" @href="#" />
```
Comment (heatherlarsen) [inline website/docs/components/toast/partials/guidelines/guidelines.md:87]: This example was changed to a Standalone Link, but the documentation above still advocates for the usage of a tertiary button. I think if they need to use a button (technically), they're stuck with the tertiary variant, so you may want to revert this change.

File: website/docs/components/alert/partials/guidelines/guidelines.md (hunk 1 above)
Comment (jorytindall) [reply website/docs/components/alert/partials/guidelines/guidelines.md:130]: Realizing in the review, the Toast documentation uses this capitalization as well.

**Review by jorytindall — APPROVED** [review]
Left just one small comment :shipit:

File: website/docs/components/toast/partials/guidelines/guidelines.md [hunk 3]
```diff
@@ -102,12 +102,15 @@ Avoid using critical Buttons in Toasts. We handle the prominence and importance
 
 ### Links
 
-Use [Standalone Links](/components/link/standalone) when an action takes the user to a new destination (URL). Follow the Standalone Link [usage guidelines](/components/link/standalone#usage) to determine what variant to use.
+Use [Standalone Links](/components/link/standalone) when an action takes the user to a new destination (URL). Follow the Standalone Link [usage guidelines](https://helios.hashicorp.design/components/link/standalone) to determine what variant to use.
```
Comment (heatherlarsen) [inline website/docs/components/toast/partials/guidelines/guidelines.md:105]: > Follow the standalone link [usage guidelines](https://helios.hashicorp.design/components/link/standalone) to determine what variant “type” to use.

Is this part still necessary since we're now providing them with an official recommendation?

**Review by heatherlarsen — COMMENTED** [review]
Looking good! Left a couple of suggestions and questions.

File: website/docs/components/alert/partials/guidelines/guidelines.md (hunk 1 above)
Comment (cveigt) [reply website/docs/components/alert/partials/guidelines/guidelines.md:130]: Let's capitalize it in both places. [Here](https://helios.hashicorp.design/components/link/standalone#when-not-to-use) we refer to the inline link as "Inline Link", for example.

File: website/docs/components/alert/partials/guidelines/guidelines.md [hunk 4]
```diff
@@ -110,7 +110,7 @@ We recommend using the `secondary` button variant for primary actions and the `t
   <A.Title>Recommended button usage</A.Title>
   <A.Description>Lorem ipsum dolar sit amet.</A.Description>
   <A.Button @text="Your action" @color="secondary" @onClick={{this.noop}} />
-  <A.Link::Standalone @icon="arrow-right" @iconPosition="trailing" @text="Another action" @href="#" />
+  <A.Link::Standalone @icon="arrow-right" @iconPosition="trailing" @color="secondary" @text="Another action" @href="#" />
```
[the remark this answers is not in the document]
Comment (cveigt) [reply website/docs/components/alert/partials/guidelines/guidelines.md:113]: Actually, it is a `Standalone Link` in production already. I'll go ahead and change it to a button. Thanks for pointing that out.

File: website/docs/components/alert/partials/guidelines/guidelines.md (hunk 1 above)
Comment (cveigt) [reply website/docs/components/alert/partials/guidelines/guidelines.md:130]: It's not necessary. I'll remove it.

File: website/docs/components/toast/partials/guidelines/guidelines.md (hunk 2 above)
Comment (cveigt) [reply website/docs/components/toast/partials/guidelines/guidelines.md:87]: Same as the `Alert`, I'll go ahead and change it to a button.

**Review by didoo — COMMENTED** [review]
A couple of points:

1) maybe this could be removed now? I don't think there are any more Structure banner in the codebase
screenshot_2797

2) I think there are still a couple of places in the Alert page where the link is primary not secondary in the page:
screenshot_2796
screenshot_2795

**Comment by cveigt** [comment]
@didoo thanks for reviewing and catching those instances. I've updated the remaining instances and removed the banner as suggested.

**Review by didoo — APPROVED** [review]
👍
